modules/integration_azure-event-hub/conf/readme.yaml

```diff
@@ -1,3 +1,13 @@
 documentations:
   - name: Azure Monitor metrics
     url: 'https://docs.microsoft.com/en-us/azure/azure-monitor/platform/metrics-supported'
+notes: |
+  ### About filtering
```

Comment:
I don't think you should add this note until you change the default filtering.

here is an example of default filtering which does not support the default tagging convention: https://github.com/claranet/terraform-signalfx-detectors/pull/258/files

of course I let you decide what kind of filter you prefer to use to replace missing custom dimensions.